redaxo/src/addons/structure/lib/linkmap/form_element/linklist.php

```diff
@@ -30,6 +30,7 @@ public function setCategoryId($categoryId)
 
     public function formatElement()
     {
+        /** @var int $widgetCounter */
         static $widgetCounter = 1;
```

Comment:
Statische Variablen benötigen immer eine `@var`-Annotation für psalm/phpstan.
Denn sonst betrachten die Tools die als mixed, weil sie ja in einem früheren Aufruf der Methode auf andere Werte gesetzt worden sein könnte.